Assets/VRTK/Scripts/VRTK_PlayerClimb.cs

```diff
@@ -240,11 +242,11 @@ private bool IsClimbableObject(GameObject obj)
             return interactObject != null && interactObject.AttachIsClimbObject();
         }
 
-        private void Update()
+        private void LateUpdate()
         {
             if (isClimbing)
             {
-                transform.position = startPosition - (GetPosition(controllerTransform) - startControllerPosition);
+                transform.position = startPosition - (GetPosition(controllerTransform) - startControllerPosition) + (climbingObject.transform.position - startClimbingObjectPosition);
```

Comment:
This is a great first pass, but you're going to run into issues when the climbing object starts rotating. You've taken translation into account here, but rotation probably needs to be added as well.